src/org/mozilla/javascript/ScriptRuntime.java

```diff
@@ -2430,13 +2430,13 @@ private static void enumChangeObject(IdEnumeration x) {
      */
     public static boolean loadFromIterable(
             Context cx, Scriptable scope, Object arg1, BiConsumer<Object, Object> setter) {
-        if ((arg1 == null) || Undefined.instance.equals(arg1)) {
+        if ((arg1 == null) || Undefined.isUndefined(arg1)) {
```

Comment:
minor: we can remove the brackets here to unify the code

Reply:
Sure, I'm not used to doing that but I'd like it to match.